GoogleMeetup/src/main/webapp/script.js

```diff
@@ -12,17 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+async function createHeatmap() {
+  var response = await fetch('/data');
+  const data = await response.json();
+  const hotspots = data.data.monitor;
+  var heatmapData = [];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const map = new google.maps.Map(document.getElementById('map'), {
+    center: new google.maps.LatLng(-33.8, 151.1),
```

Comment:
Do we need this? Won't it auto-center on the parts of the heatmap that have some intensity?

Reply:
Apparently not. Tried to render it without the center and just wouldn't render the map at all.